### DISCUSSION TITLE
Update Thumbnail of Sandbox

### DISCUSSION THREAD

**Discussion by jrock2004** [codesandbox/codesandbox-client#6472] [General]
When I look at my dashboard the thumbnail is an old error page instead of something else. I saw in a closed issue that you could call an API to update your screenshot, but not sure how. Should this not be something the UI could do for us?

![image](https://user-images.githubusercontent.com/655716/154189322-5dcad8e7-589d-497a-9931-2b0739849fa4.png)

**Comment by JamesACS** [comment]
@jrock2004 - Out of interest, does the image refresh if you make some alterations to the Sandbox and reload it? 

As a workaround you can also take a screencap of the project, name it thumbnail.png and upload it - the preview should then show that image

**Reply by sannek** [reply]
@jrock2004  - not to hijack James' answer but it is indeed possible to regenerate the screenshot:

Making a `GET` to `codesandbox.io//api/v1/sandboxes/:sandbox_id/screenshot.png` should regenerate it, where `:shortid` is the 5-6 character random id for your sandbox. The easiest way to find it is to open the sandbox, and the preview will have `shortid.csb.app` as the url.

(I'll also have a look at when / how we actually trigger updating the screenshot, because it should probably be regenerated occasionally if changes have been made)
